regolith/credentials2.json

```diff
@@ -0,0 +1,10 @@
+{"installed":
```

Comment:
This needs to be removed.  In fact we should kill this branch and put everything on a new branch because you just put all your secrets for accessing your Google account onto a public repo.....

Reply:
I am a bit confused about this part. Should I always leave the credentials file uncommitted when submitting the changes? Additionally, is leaving it uncommitted okay if the script uses the file?

Reply:
we may also want to renew all the credentials and perhaps the link to the group Gdoc directory.  Please also take this opportunity to reread the group security wiki that you read during onboarding.   Do we need to put this issue in there explicitly if it is not there?  Would you do that to help future folks?